judge/models/problem.py

```diff
@@ -442,6 +442,25 @@ def save(self, *args, **kwargs):
 
     save.alters_data = True
 
+    def can_vote(self, user):
```

Comment:
Now that this PR is almost ready for merging, could we add some tests for this function? You don't need to test every single branch.

Reply:
I kind of did a test for everything when I merged the separate functions into `can_vote()`. Everything seems to work as intended.

Reply:
What I meant is add some unittests in https://github.com/DMOJ/online-judge/blob/master/judge/models/tests/test_problem.py#L11.

Reply:
I've added unit tests for can_vote. Is there anything else they are needed for?

Reply:
Probably good enough.